addons/mrp/mrp.py

```diff
@@ -151,6 +151,10 @@ class mrp_bom(osv.osv):
     _description = 'Bill of Material'
     _inherit = ['mail.thread']
 
+    def _get_bom_ids(self, cr, uid, product_template_ids, context=None):
```

Comment:
I would rename to _get_bom_ids_from_product_template_ids. Indeed this method is only used in a store dict, and the name should be quite precise to avoid confusion.